webpack/set-webpack-public-path-plugin/src/SetPublicPathPlugin.ts

```diff
@@ -130,13 +125,17 @@ const ASSET_NAME_TOKEN: string = '-ASSET-NAME-c0ef4f86-b570-44d3-b210-4428c5b782
 
 const ASSET_NAME_TOKEN_REGEX: RegExp = new RegExp(ASSET_NAME_TOKEN);
 
+function escapeRegExp(str: string): string {
+  return str.replace(/[.*+?^${}()|[\]\\]/g, '\\$&');
```

Comment:
Why not the version from the actual package? `/[\\^$.*+?()[\]{}|]/g`

Reply:
Honestly I think it's the same, just in a different order. Feel free to ignore.

Reply:
This is what GitHub Copilot suggested. They're equivalent, so I'll just leave this as-is.